crates/fuel-streams-core/src/nats/nats_client.rs

```diff
@@ -64,46 +51,39 @@ impl NatsClient {
         Ok(self)
     }
 
-    pub async fn create_stream(
+    pub async fn create_store(
```

Comment:
This makes sense since it's a Nats concern. to even make it more specific, we can rename as: `create_kv_store`